### PR TITLE
Migration to React 18

### PR DESCRIPTION
**Pull request by buberdds**
- cannot update all React related deps due to issues with `webext-redux`
- entry in `resolutions` is needed to avoid `act` warnings in tests when using userEvents
- there are still 3 warnings in loadable tests, but I have no idea how to avoid this, because warnings will be thrown when suspended data is not resolved and these tests cover such case
- `importAccountsActions.clear()` action had to be moved to saga. This UI part was not working with a new React 18 strict mode

### CODE REVIEW

File: src/utils/__tests__/loadable.test.tsx [hunk 1]
```diff
@@ -42,10 +42,9 @@ describe('loadable', () => {
   })
```
Comment (lukaw3d) [inline src/utils/__tests__/loadable.test.tsx:42]: Oddly this works:
```tsx
  it('should render fallback if given one', async () => {
    const component = render(<LazyComponentWithFallback />)
    expect(await screen.findByText(/lazy-loaded/)).toBeInTheDocument()
    expect(component.container.firstChild).toMatchSnapshot()
  })
```
but this complains `Warning: A suspended resource finished loading inside a test, but the event was not wrapped in act(...).`:
```tsx
  it('should render fallback if given one', async () => {
    const component = render(<LazyComponentWithFallback />)
    expect(await screen.findByText(/Loading/)).toBeInTheDocument()
    expect(component.container.firstChild).toMatchSnapshot()
    expect(await screen.findByText(/lazy-loaded/)).toBeInTheDocument()
    expect(component.container.firstChild).toMatchSnapshot()
  })
```

File: src/app/state/wallet/saga.ts [hunk 2]
```diff
@@ -77,7 +78,7 @@ export function* openWalletFromPrivateKey({ payload: privateKey }: PayloadAction
   const walletAddress = yield* call(publicKeyToAddress, publicKeyBytes)
   const publicKey = uint2hex(publicKeyBytes)
   const balance = yield* call(getBalance, publicKeyBytes)
-
+  yield* put(importAccountsActions.clear())
```
Comment (lukaw3d) [inline src/app/state/wallet/saga.ts:81]: Please split into a separate commit

Comment (buberdds) [reply src/app/state/wallet/saga.ts]: done

**Review by lukaw3d — APPROVED** [review]
